test/Api/CatalogFixtures.php

```diff
@@ -0,0 +1,245 @@
+<?php
+namespace SquareConnect\Api;
+
+class CatalogFixtures extends \PHPUnit_Framework_TestCase
```

Comment:
because this class is not testing anything. It's just a helper class that builds objects.